browser/components/loop/content/css/panel.css

```diff
@@ -67,10 +82,16 @@ body {
 /* Tabs and tab selection buttons */
 
 .tab-view-container {
+  width: 100%;
   background-image: url("../shared/img/beta-ribbon.svg#beta-ribbon");
   background-color: #fbfbfb;
   background-size: 36px 36px;
   background-repeat: no-repeat;
+  flex: 1;
+  display: flex;
+  flex-flow: column nowrap;
+  height: 368px;
```

Comment:
Removed the height and tested in panel.  Inspected chrome computed height and gave correct adjusted heights. Done.